### PR TITLE
fix: inline text link state

### LINKED ISSUE #186: [request]: Inline text links need a new state
## 🚀 Feature Proposal
Currently inline text links only show an underline on hover. This makes it hard to realise what is and isnt a link and impossible to see on mobile. 

Perhaps the link should remain constantly underlined and we introduce a new state on hover? Perhaps a 50% opactiy.

See below for current functionality vs proposed.

**Current.**

https://user-images.githubusercontent.com/85284662/169243566-ed1a6608-773b-47d7-9dfa-a4396c2f1282.mov

**Proposed**

https://user-images.githubusercontent.com/85284662/169243614-8b409147-bb99-4491-815c-ad8397406ab5.mov

### PR DESCRIPTION
**Pull request by elenamarinaki**
### Why

Resolves #186 

### What

- applied inline css to the `InlineLink` prop
- removed heading class that was overriding the above rule

### Checklist

- [ ] Ready to be merged

### CODE REVIEW

File: packages/web/src/client/components/Renderer/RendererRichText.tsx [hunk 1]
```diff
@@ -160,7 +160,16 @@ const components: Partial<PortableTextReactComponents> = {
     ) => {
       return isArrayGuard(props.children) &&
         isStringGuard(props.children[0]) ? (
-        <InlineLink {...props.value} label={props.children[0]} />
+        <InlineLink
+          {...props.value}
+          label={props.children[0]}
+          css={{
```
Comment (joshuaellis) [inline packages/web/src/client/components/Renderer/RendererRichText.tsx:166]: Why did you decide to put the styling here and not on the element?

File: packages/web/src/client/components/Links/InlineLink.tsx [hunk 2]
```diff
@@ -23,9 +23,4 @@ const Anchor = styled(LinkBase, {
   fontWeight: 'inherit',
   lineHeight: 'inherit',
   fontFamily: 'inherit',
-  textDecoration: 'none',
```
Comment (joshuaellis) [inline packages/web/src/client/components/Links/InlineLink.tsx:26]: maybe these should be `inherit` like the above? What do you think?

**Review by joshuaellis — CHANGES_REQUESTED** [review]

File: packages/web/src/client/components/Links/InlineLink.tsx (hunk 2 above)
Comment (elenamarinaki) [reply packages/web/src/client/components/Links/InlineLink.tsx:26]: @joshuaellis 

how would that work? so the new styling goes inside the `LinkBase` and the text decoration here should be `inherit`?
(I tried that btw, doesn't seem to work)

Comment (joshuaellis) [reply packages/web/src/client/components/Links/InlineLink.tsx:26]: No i just think set it as inherit incase we set it elsewhere.

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**